client/assets/content_bundles/home_index.en.yaml

```diff
@@ -11,16 +11,14 @@ contents:
       image_name: undraw-home-promo-symptoms
   items:
     - item_type: recent_numbers
-      title: Recent Numbers
-      button_text: "See All ›"
+      title: Recent Numbers ›
       href: /recent-numbers
     - item_type: protect_yourself
-      title: Protect Yourself
-      button_text: "Learn More ›"
+      title: Protect Yourself ›
       href: /protect-yourself
     - item_type: information_card
       title: Get the Facts
       subtitle: Spraying alcohol or chlorine all over your body does not kill the new coronavirus.
       href: /get-the-facts
       button_text: Learn More
-      image_name: streamline-gtf-spray
+      image_name: streamline-gtf-spray
```

Comment:
You shouldn't do a whitespace only change. I think it's convention to have trailing newlines in our code but I'm not sure if that's formalized in a style guide. It's changes like this that cause a lot of churn between engineers with different style. The best practice is to leave the style as is unless you've got a good reason to change it.... e.g. adhering to formal style guidelines.